thunder.profile

```diff
@@ -239,12 +239,54 @@ function thunder_themes_installed($theme_list) {
   }
 }
 
+/**
+ * Helper function to check if all required modules are available on install.
+ *
+ * @param array $installed_modules
+ *   Modules installed.
+ * @param array $required_modules
+ *   Required modules to be available.
+ * @param bool $only_direct_module_installs
+ *   The flag to only check if direct module install is executed.
+ *
+ * @return bool
+ *   Returns if on module install requirements are fulfilled.
+ */
+function _thunder_are_modules_installed(array $installed_modules, array $required_modules, $only_direct_module_installs = TRUE) {
+  // Check direct module installs.
+  if (
+    $only_direct_module_installs
+    && (drupal_installation_attempted() || Drupal::isConfigSyncing())
+  ) {
+    return FALSE;
+  }
+
+  // Check that at least one required module is in list of installed modules.
+  $required_not_installed_modules = array_diff($required_modules, $installed_modules);
```

Comment:
I think $installed_modules name is confusing, because it passes a list of modules currently installing, not the ones installed in the system, which are called active_modules below. 
hook_modules_installed() says
> An array of the modules that were installed.